production/helm/README.md

```diff
@@ -10,6 +10,12 @@ Loki's chart repository to Helm:
 $ helm repo add loki https://grafana.github.io/loki/charts
 ```
 
+If you have added it, need to update in case miss any sub-repo:
```

Comment:
This sentence doesn't make any sense. Should be something more like `You can update the chart repository by running:`. Though I think anyone using helm would know such info...

Reply:
Not resolved